src/test/java/htsjdk/samtools/CRAMFileWriterTest.java

```diff
@@ -26,28 +26,40 @@
 import htsjdk.HtsjdkTest;
 import htsjdk.samtools.cram.ref.ReferenceSource;
 import htsjdk.samtools.reference.InMemoryReferenceSequenceFile;
+import htsjdk.samtools.reference.ReferenceSequenceFileFactory;
+import htsjdk.samtools.util.IOUtil;
 import htsjdk.samtools.util.Log;
 import htsjdk.samtools.util.Log.LogLevel;
 import org.testng.Assert;
+import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Collections;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class CRAMFileWriterTest extends HtsjdkTest {
 
+    final LogLevel globalLogLevel = Log.getGlobalLogLevel();
+    final File SAM_TOOLS_TEST_DIR = new File("src/test/resources/htsjdk/samtools");
+
     @BeforeClass
     public void initClass() {
         Log.setGlobalLogLevel(LogLevel.ERROR);
```

Comment:
not sure I understand what you mean...I want to make sure that when the test is over the logging level goes back to what it was...otherwise it will be set for all the remaining tests..

Reply:
I meant why not save the log level to at this point rather than on class initialization.

Reply:
I see....so leave the `finit()` method, but replace the `init()` method with variable initiation?

Reply:
oh...I get it. sorry....I'm a bit thick 😸